src/components/ObsDetails/ActivityItem.js

```diff
@@ -41,34 +44,36 @@ const ActivityItem = ( {
   );
 
   return (
-    <View className={item.temporary && "opacity-50"}>
+    <View className="flex-column mx-3">
       <ActivityHeader
         item={item}
         refetchRemoteObservation={refetchRemoteObservation}
         toggleRefetch={toggleRefetch}
       />
       {taxon && (
-        <Pressable
-          className="flex-row my-3 ml-3 items-center"
-          onPress={navToTaxonDetails}
-          accessibilityRole="link"
-          accessibilityLabel={t( "Navigate-to-taxon-details" )}
-        >
-          {isOnline
-            ? <TaxonImage uri={Taxon.uri( taxon )} />
-            : showNoInternetIcon( t( "Taxon-photo-unavailable-without-internet" ) )}
-          <View>
-            <Text className="text-lg">{taxon.preferred_common_name}</Text>
-            <Text className="color-darkGray">
-              {taxon.rank}
-              {" "}
-              {taxon.name}
-            </Text>
-          </View>
-        </Pressable>
+        <View className="flex-row items-center justify-between">
+          <Pressable
+            className="flex-row my-3 ml-3 items-center"
+            onPress={navToTaxonDetails}
+            accessibilityRole="link"
+            accessibilityLabel={t( "Navigate-to-taxon-details" )}
+          >
+            {isOnline
+              ? <TaxonImage uri={Taxon.uri( taxon )} />
+              : showNoInternetIcon( t( "Taxon-photo-unavailable-without-internet" ) )}
+            <DisplayTaxonName scientificNameFirst={false} taxon={taxon} layout="vertical" />
+          </Pressable>
+          <Pressable
```

Comment:
This is what I see. It seems like the button is showing for me. Is this the same situation where you are not seeing the button? This brought up another question I had. Should the agree button be shown when it is your own identification? 
<img width="344" alt="Screen Shot 2023-04-02 at 5 14 17 PM" src="https://user-images.githubusercontent.com/13282161/229386842-2fadea2f-a8e1-4cac-b7ce-93a1552ac50c.png">

Reply:
Oh got it, didn't recognize this as the agree button because it is showing on every activity item. For questions like these, I'd recommend taking a look at what we're doing on the website, since we're porting this functionality over to the mobile app.

[Here's](https://github.com/inaturalist/inaturalist/blob/cf79c93a0f6ddb93196d3ae5b922dc01392c3ae1/app/webpack/observations/identify/components/observations_grid_item_for_identify.jsx#L43) where that code lives on the web.

So it looks like we're showing an agree button only at certain rank levels & when the identifier is not the current user (and when there is an identification, not simply a comment).